galaxies/galaxiesModel.spec.js

```diff
@@ -0,0 +1,60 @@
+const Galaxies = require('./galaxiesModel');
+const db = require('../data/dbConfig');
+
+describe('The Galaxies Model', () => {
+
+  beforeEach(() => {
+    return db('galaxies').truncate();
+  });
+
+  describe('The Insert Function', () => {
+
+    it('should insert a galaxy into the db', async () => {
+      await Galaxies.insert({name: 'sombrero'});
+
+      const galaxies = await db('galaxies');
+
+      expect(galaxies.length).toBe(1);
+      expect(galaxies[0].name).toBe('sombrero');
+    });
+
+    it('should return the inserted galaxy with id', async () => {
+      const galaxy = await Galaxies.insert({name: 'sunflower'});
+
+      expect(galaxy.id).toBe(1);
+      expect(galaxy.name).toBe('sunflower');
+    });
+  });
+
+  describe('The GetAll Function', () => {
+
+    it('should retreive all galaxies in db', async () => {
+      await db('galaxies').insert([
+        {name: 'sombrero'},
+        {name: 'sunflower'}
+      ]);
+
+      const galaxies = await Galaxies.getAll();
+
+      expect(galaxies.length).toBe(2);
+      expect(galaxies[0].name).toBe('sombrero');
+    });
+  });
+
+  describe('The Remove Function', () => {
+
+    it('should delete a galay by id', async () => {
+      await db('galaxies').insert([
+        {name: 'sombrero'},
+        {name: 'sunflower'}
+      ]);
```

Comment:
You could move a bit of this out into an outer describe for this one and the one before it. For example:
```javascript
describe('', () => {
  beforeEach(async () => {
    await db('galaxies').insert([
      {name: 'sombrero'},
      {name: 'sunflower'}
    ]);
  });

  describe('The GetAll Function', () => {
    it('should retreive all galaxies in db', async () => {
      const galaxies = await Galaxies.getAll();

      expect(galaxies.length).toBe(2);
      expect(galaxies[0].name).toBe('sombrero');
    });
  });

  describe('The Remove Function', () => {
    it('should delete a galay by id', async () => {
      await Galaxies.remove(1);

      const galaxies = await Galaxies.getAll();

      expect(galaxies.length).toBe(1);
      expect(galaxies[0].name).toBe('sunflower');
    });
  });
});
```